sourcetracker/_sourcetracker.py

```diff
@@ -618,296 +626,105 @@ def gibbs_sampler(cp, sink, restarts, draws_per_restart, burnin, delay):
     return mixing_proportions, calculated_assignments
```

Comment:
Better description for "calculated_assignments" - this is analogous to the "full_results"?